src/js/utils/specs/string-utils-spec.js

```diff
@@ -27,6 +27,10 @@ define(
                         'Hello, {name}. Here is a { followed by a }', {name: 'Andy'},
                         'Hello, Andy. Here is a { followed by a }'
                     ]
```

Comment:
Missing a comma.
Maybe this is breaking the build? The error reported in Travis is not helpful.  I re-ran master and it is passing, so hopefully this is it.

Reply:
swalladge: Maybe you missed the first line above?  There is a bug.  You are missing a comma at the end of line 29.

Reply:
Thanks, nice catch. 

The unhelpful travis error is due to recent changes with how xvfb works with travis (the previous builds including the one you reran were on trusty, travis now defaults to xenial). I'm going to try using the latest method for xvfb with xenial, otherwise will explicitly pin back to trusty and use the original method.

Reply:
Tests passing now, i'll squash in prep for merge. Thanks for your review @robrap ! :)